hw_5/homework5.py

```diff
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, Request
+from fastapi.templating import Jinja2Templates
+from fastapi.responses import HTMLResponse
+from pydantic import BaseModel
+import pandas as pd
+
+
+app = FastAPI()
+templates = Jinja2Templates(directory="templates")
+tasks = []
+
+class Task(BaseModel):
+    id: int
+    head: str
+    description: str
+    status: str
+
+
+@app.get("/tasks/", response_class=HTMLResponse)
+async def show_task(request: Request):
+    table = pd.DataFrame([vars(task) for task in tasks]).to_html()
+    return templates.TemplateResponse("task.html", {"request": request, "table": table})
+
+
+@app.get("/tasks/{id}")
```

Comment:
Отображается неверно, так как не указан тип ответа функции
```suggestion
@app.get("/tasks/{id}", response_class=HTMLResponse)
```